selfhost/typechecker.jakt

```diff
@@ -2896,7 +2914,7 @@ struct Typechecker {
                                 "Type mismatch: expected ‘{}’, but got ‘{}’"
                                 .type_name(TypeId::from_string(seen_type_id_string.value()))
                                 .type_name(rhs_type_id)
-                            )
+                            ),
```

Comment:
Unrelated change. Also comma is not required here.
There are a whole bunch of unrelated changes like this in the commit, please remove them :)